upup/models/bindata.go

```diff
@@ -4587,7 +4587,7 @@ spec:
               key: wait-bpf-mount
               name: cilium-config
               optional: true
-        image: "docker.io/cilium/cilium:{{- or .Version "v1.7.3" }}"
+        image: "docker.io/cilium/cilium:{{ "v1.7.3" }}"
```

Comment:
also wondering about this version. it'd be nice if we could reduce the number of places  these are defined.

Reply:
(i really should be commenting on the yaml but bindata.go sorts first when viewing GH diffs so I see this before the yaml... ☹️  )

Reply:
Yep. I am working on a follow-up PR to clean up these templates.